lib/bundle/extend/os/linux/skipper.rb

```diff
@@ -4,7 +4,10 @@ module Bundle
   module Skipper
     class << self
       def skip?(entry, silent: false)
-        return generic_skip?(entry) unless [:cask, :mas].include?(entry.type)
+        unless [:cask, :mas].include?(entry.type) ||
```

Comment:
I find this a little hard to grok and needed to read it a couple of times before understanding what we're aiming for here. Could we use some methods to wrap these to better convey what we're doing?

```rb
unless skippable_plugins?(entry) || cask_tap?(entry)
  return generic_skip?(entry)
end

# ...

def skippable_plugins?(entry)
  [:cask, :mas].include?(entry.type)
end

def cask_tap?(entry)
  entry.type == :tap && entry.name == "homebrew/cask"
end
```

I don't love the `skippable_plugins` method name here but just to demonstrate the idea. Not a deal breaker though if others find this readable enough.

Reply:
I considered this, but couldn't think of any decent method names. :grinning: I agree it makes more sense when split up - but I wonder if the `unless` is the thing that's really confusing us?